pkg/resource/caconfig.go

```diff
@@ -118,6 +118,17 @@ func (gcac *generatorCAConfig) expected() (runtime.Object, error) {
 		}
 	}
 
+	cp_ca, err := gcac.openshiftConfigLister.Get("cloud-provider-config")
+	if errors.IsNotFound(err) {
+		klog.Infof("missing the cloud-provider-config configmap: %s", err)
+	} else if err != nil {
+		return cm, err
+	} else {
+		if cert, ok := cp_ca.Data["ca-bundle.pem"]; ok {
+			cm.Data["cloud-provider-ca-bundle.pem"] = cert
```

Comment:
This config map is used by the nodeca daemon and is distributed to all nodes to /etc/docker/certs.d. Therefore its keys should be host names.

Perhaps you need yet another config map to this CA.

Reply:
Let's track this in https://bugzilla.redhat.com/show_bug.cgi?id=1794690